src/gbt_model_builder.pyx

```diff
@@ -179,3 +179,33 @@ def gbt_reg_model_builder(n_features, n_iterations):
     :param size_t n_iterations: Number of trees in the model
     '''
     return gbt_regression_model_builder(n_features, n_iterations)
+
+class GBTModel():
+    '''
+    Daal4py GBT Model
+    '''
+
+    def __init__(self, model):
+        framework_name = (model.__class__.__module__, model.__class__.__name__)
```

Comment:
It doesn't look like framework name, better rename to `submodule_name` and `class_name`